scripts/dynatrace_ingestion/ingest.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+if [ "$#" -ne 3 ]; then
```

Comment:
a small readme around this with what it is for, how to setup and how to excersice it would be great. Also, was this used to generate any of the reference data? If so it makes sense to describe how that reference was created through this.

Reply:
i now see the additions to the readme, maybe expand a bit on that.

Reply:
@craffit thanks for the tip, I pushed a change to the README 👍